modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -16,7 +16,7 @@ You can use the `oc adm must-gather` CLI command to collect information about yo
 
 * The {product-title} CLI (`oc`) installed.
 
-.Precedure
+.Procedure
 
 . To collect {SMProductName} data with `must-gather`, you must specify the {SMProductName} image.
```

Comment:
Not going to hold this PR up but adding a descriptor after `must-gather` could add clarity. I _think_ `must-gather` command would be right.